src/include/stir/recon_buildblock/BinNormalisationWithCalibration.h

```diff
@@ -0,0 +1,65 @@
+//
+//
+/*
+    Copyright (C) 2020, UCL
+    Copyright (C) 2020, NPL
+    See STIR/LICENSE.txt for details
+*/
+/*!
+  \file
+  \ingroup normalisation
+
+  \brief Declaration of class stir::BinNormalisationWithCalibration
+
+  \author Daniel Deidda
+  \author Kris Thielemans
+*/
+
+#ifndef __stir_recon_buildblock_BinNormalisationWithCalibration_H__
+#define __stir_recon_buildblock_BinNormalisationWithCalibration_H__
+
+
+#include "stir/RegisteredParsingObject.h"
+#include "stir/Bin.h"
+#include "stir/shared_ptr.h"
+#include "stir/recon_buildblock/BinNormalisation.h"
+#include "stir/decay_correction_factor.h"
+
+START_NAMESPACE_STIR
+
+/*!
+  \ingroup normalisation
+*/
+class BinNormalisationWithCalibration : 
+        public RegisteredParsingObject<BinNormalisationWithCalibration, BinNormalisation >
```

Comment:
```suggestion
        public BinNormalisation
```
only (final) leaf classes should be derived from `RegisteredParsingObject`. Changing this should resolve your error removing `registered_name`